### PR TITLE
docs: add default channel example

### PR DESCRIPTION
**Pull request by robszumski**
@marineam Does this cover everything we were talking about?

### CODE REVIEW

File: Documentation/configure-machines.md [hunk 1]
```diff
@@ -4,7 +4,34 @@ Configuring new or existing CoreOS machines to communicate with a [CoreUpdate](h
 
 ## New Machines
 
-New servers can be configured to communicate with your CoreUpdate installation by using [cloud-config](https://coreos.com/docs/cluster-management/setup/cloudinit-cloud-config). Set the value of `server` to the custom address of your installation and `group` to the unique identifier of your application group.
+New servers can be configured to communicate with your CoreUpdate installation by using [cloud-config](https://coreos.com/docs/cluster-management/setup/cloudinit-cloud-config).
+
+By default, your installation has a single application, CoreOS, with the identifier `e96281a6-d1af-4bde-9a0a-97b76e56dc57`. This ID is universal and all CoreOS machines are configured to use it. Within the CoreOS application, there are several application groups which have been created to match CoreOS channels (`alpha`, `beta`, etc). Each of these groups have a unique identifier that is a generated UUID or custom string, which must be configured on your CoreOS machines.
```
Comment (marineam) [inline Documentation/configure-machines.md]: This sounds like the default groups use UUID but they don't, it is alpha, beta or stable. Only custom groups would have a UUID.

File: Documentation/configure-machines.md [hunk 2]
```diff
@@ -38,6 +65,14 @@ To apply the changes, run:
 sudo systemctl restart update-engine
 ```
 
+In addition to `GROUP=` and `SERVER=`,  a few other options exist but are set to defaults:
```
Comment (marineam) [inline Documentation/configure-machines.md]: Lets call them internal values instead of options

**Comment by robszumski** [comment]
Updated, PTAL

**Comment by sym3tri** [comment]
LGTM

**Pull request merged**